.github/workflows/node.js.yml

```diff
@@ -0,0 +1,26 @@
+name: Node.js CI
+
+on:
+  pull_request:
+    branches: 
+      - "development"
+
+jobs:
+  type-check-lint-test-build:
+    name: type-check-lint-test-build
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js 18.x
+      uses: actions/setup-node@v3
+      with:
+        node-version: "18.x"
+        cache: 'npm'
+    - uses: nrwl/nx-set-shas@v3
+    - run: npm install nx
+    - run: npm install
+    - run: npm run typecheck
+    - run: npm run lint
+    - run: npm run test
+    - run: npm run build --if-present
```

Comment:
why did you add the --if-present flag. I think it is an option for scripts that may not exist in scripts section which is not the case here. Do I miss sth?